frontend/src/components/DefenceBox/DefenceBox.jsx

```diff
@@ -17,6 +17,13 @@ function DefenceBox(props) {
       isActive: false,
       isTriggered: false,
     },
+    {
+      name: "random sequence enclosure",
+      id: "RANDOM_SEQUENCE_ENCLOSURE",
+      info: "enclose the prompt between a random string and instruct bot to only follow. this is a form of prompt validation.", 
```

Comment:
Is this an incomplete sentence here? I don't quite understand the 'to only follow' bit.